usaspending_api/etl/management/commands/load_submission.py

```diff
@@ -643,9 +711,8 @@ def load_file_c(submission_attributes, db_cursor, award_financial_frame):
         }
 
         # individual FILE C D linkage
```

Comment:
move line 713 to 733 to inside new get or create summary award.  this way we dont have to do 2 db calls